python/foglamp/services/core/api/configuration.py

```diff
@@ -511,7 +511,32 @@
     except ValueError as ex:
         raise web.HTTPNotFound(reason=str(ex))
 
-    return web.json_response({"message": "Parent-child relationship for the parent-{} is deleted".format(category_name)})
+    return web.json_response({"message": "Parent category-{} is deleted".format(category_name)})
```

Comment:
Restore this message. This is just an old def with incorrect name. i.e. delete_parent_category > delete_parent_relation()